### ISSUE TITLE
Versioninformation.ts not found

### ISSUE THREAD

**Issue by kleinech** [ultimate-comparisons/ultimate-comparison-BASE#107]
Updating a comparison with the update branch and run command `npm start` causes error:

```
ERROR in ./app/components/comparison/components/comparison.component.ts
Module not found: Error: Cannot resolve 'file' or 'directory' ../../../VersionInformation in /home/kleinech/Projekte/ultimate-comparisons/ultimate-reference-management-software-comparison/app/components/comparison/components
 @ ./app/components/comparison/components/comparison.component.ts 17:29-67

ERROR in [default] /home/kleinech/Projekte/ultimate-comparisons/ultimate-reference-management-software-comparison/app/components/comparison/components/comparison.component.ts:7:35 

```

**Comment by hueneburg** [comment]
Could not reproduce...
Taken steps:
1. clone into new repo
2. check out random commit in master branch (I chose c5d47de)
3. `npm install`
4. `npm start`
5. `git merge --allow-unrelated-histories -X theirs origin/update`
6. delete gulpfile.js in favor of gulpfile.babel.js
7. `npm install`
8. `npm start`

If I should have missed something, please tell me.

**Comment by kleinech** [comment]
Steps to reproduces:
1. `git clone https://github.com/ultimate-comparisons/ultimate-reference-management-software-comparison`
2. `git remote add base https://github.com/ultimate-comparisons/ultimate-comparison-BASE.git`
3. `git pull -Xtheirs base update`
4. ` npm install`
5. `npm start`
6. Error

**Comment by kleinech** [comment]
Possible issue:
File VersionInformation.ts MUST be created by user?

**Comment by hueneburg** [comment]
OK, for some reason VersionInformation.ts is not in the update branch... I'm guessing that `git add --all` does not include it (for some reason)...

**Comment by kleinech** [comment]
Manually added Version.ts to update branch

**Comment by koppor** [comment]
May I assume that it is regenerated when updating the update branch? Example: Version 1.0.0 is released then Version 1.1.0. Does `VersionInformation.ts` in the update branch still contain "1.0.0"?
